### PR TITLE
Add features for #9 and #10

### PR DESCRIPTION
**Pull request by thomasf**
I think it's done now.. **~~I haven't tested  actually running with changes yet~~**.. 
I'll notify you when I have used these changes for a bit in the coming hours.

### CODE REVIEW

**Comment by thomasf** [comment]
I have tested all combinations now ant everything seems to work.. 

One thing tough, if an draft release exists a new one is created instead of updating an existing draft.. I havent used github releases before but to me that seems to be the way I would want it to happen so that I can choose any of the tag release drafts  to be published..

File: main.go [hunk 1]
```diff
@@ -129,34 +161,58 @@ func buildRelease(client *github.Client, owner string, repo string, tag string)
...
+		for _, asset := range assets {
+			if *asset.Name == path.Base(file) {
+				switch rc.FileExists {
+				case "overwrite":
+					// do nothing
+				case "fail":
+					return fmt.Errorf("Asset file %s already exists", path.Base(file))
+				case "skip":
+					fmt.Printf("Skipping pre-existing %s artifact\n", *asset.Name)
+					continue files
```
Comment (tboerger) [inline main.go]: No need for this goto, you just need to skip this file :)

**Comment by tboerger** [comment]
One smaller comment, otherwise LGTM

File: main.go (hunk 1 above)
Comment (thomasf) [inline main.go]: Without that goto the loop will complete and the  file will be added at the `uploadFiles = append(uploadFiles, file)` line.
What do you suggest instead, using a `found` variable and check that after the loop has been completed?

Comment (thomasf) [inline main.go]: You know,, this isnt an uncommon pattern in go...

https://github.com/golang/go/blob/08594ac7c7e123d4aa46f60690da0a7e4034f4e9/src/runtime/trace.go#L694

https://github.com/golang/go/blob/33a9a98e4d5893699749f75334c651b2adcecfb9/src/go/types/expr.go#L1056

...

Comment (thomasf) [inline main.go]: https://golang.org/doc/effective_go.html#if

>  In the Go libraries, you'll find that when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted.

**Comment by jackspirou** [comment]
LGTM

**Pull request merged**